src/modules/Header/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { LocationBadge } from './components';
```

Comment:
so, in such app size Header can be a module, but in bigger apps Header is a part of Layout.